.ci-operator.yaml

```diff
@@ -1,4 +1,4 @@
 build_root_image:
   name: release
   namespace: openshift
-  tag: rhel-8-release-golang-1.20-openshift-4.15
+  tag: rhel-8-release-golang-1.20-openshift-4.16
```

Comment:
This PR is updating the builder image to one for OpenShift 4.16 but still based on RHEL 8, so it is safe to merge.